### DISCUSSION TITLE
How to create new datatype?

### DISCUSSION THREAD

**Discussion by phenomenia** [thedevdojo/wave#120] [Q&A] [answered]
Is there a documentation how to create a new datatype?
I need a new API endpoint that returns data based on data I provide in the get request.
I found the DB table data_types but I'm not sure what else is required to process the provided data from my get request and how to return data.

**Comment by bobbyiliev** [comment, marked as answer]
Hi there,

The data_types table is utilized from the Voyager admin panel. I don't they there is a documentation on how to do that, but here is a link to a SO discussion that covers this:

> https://stackoverflow.com/questions/71921096/how-to-add-custom-column-in-voyager-browse-blade

Feel free to share more details on the data type that you want to add and I can try to advise you furhter.
